Automattic/wp-calypso
# Migrate vertical menu styles

#### Changes proposed in this Pull Request

Migrate styles for Vertical Menu Styles

#### Testing instructions

Go to http://calypso.localhost:3000/devdocs/design/vertical-menu and verify that the vertical menu styles exist.

Bad:
<img width="303" alt="vm-bad" src="https://user-images.githubusercontent.com/10561050/46320430-e9dab100-c610-11e8-89ff-dac0954569e7.png">

Good:
<img width="306" alt="vm-good" src="https://user-images.githubusercontent.com/10561050/46320416-e0e9df80-c610-11e8-80d7-8b56c86c8e87.png">

#27515
